packages/lesswrong/lib/scoring.ts

```diff
@@ -3,15 +3,35 @@ import { DatabasePublicSetting } from './publicSettings';
 const timeDecayFactorSetting = new DatabasePublicSetting<number>('timeDecayFactor', 1.15)
 const frontpageBonusSetting = new DatabasePublicSetting<number>('frontpageScoreBonus', 10)
 const curatedBonusSetting = new DatabasePublicSetting<number>('curatedScoreBonus', 10)
+const subforumCommentBoostSetting = new DatabasePublicSetting<number>('subforumCommentScoreBonus', 20);
+const subforumCommentBoostHoursSetting = new DatabasePublicSetting<number>('subforumCommentScoreBonusHours', 8);
 
 export const TIME_DECAY_FACTOR = timeDecayFactorSetting;
 // Basescore bonuses for various categories
 export const FRONTPAGE_BONUS = frontpageBonusSetting;
 export const CURATED_BONUS = curatedBonusSetting;
 
+const getSubforumScoreBoost = () => {
+  return {
+    magnitude: subforumCommentBoostSetting.get(),
+    durationHours: subforumCommentBoostHoursSetting.get(),
+  };
+}
+
+const getSubforumCommentBonus = (item: VoteableType) => {
+  if ("tagCommentType" in item && item["tagCommentType"] === "SUBFORUM") {
+    const {magnitude, durationHours} = getSubforumScoreBoost();
+    const createdAt = (item as any).createdAt ?? new Date();
+    const ageHours = (Date.now() - createdAt.getTime()) / 3600000;
+    const factor = -magnitude / (durationHours * durationHours);
+    const bias = factor * (ageHours - durationHours) * (ageHours + durationHours);
```

Comment:
```suggestion
    const bias = magnitude * (1 - (ageHours / durationHours) ** 2);
```
Slightly more readable version

Reply:
```suggestion
    const bias = magnitude * (1 - (ageHours / durationHours) ** exponent);
```
Also, rogue suggestion: we could make the exponent a database setting to tweak the recency dropoff. I have a vague idea that using a very high magnitude (like 100) with a steep dropoff (like 0.3, so the bonus will drop to 20 withing 4 hours) might be what we want. This might be a pain with the mongo version, although mongo does have a `$pow` operator so it should be possible to make it work

Reply:
(we don't need to do this before merging though)